includes/discovery/discovery-protocols.inc.php

```diff
@@ -157,6 +157,10 @@
     d_echo($lldp_array);
     $dot1d_array = snmpwalk_cache_oid($device, 'dot1dBasePortIfIndex', array(), 'BRIDGE-MIB');
     d_echo($dot1d_array);
+    if ($config['discovery_by_ip'] == true) {
+        $ptopo_array = snmpwalk_cache_oid($device, 'ptopoConnEntry', array(), 'PTOPO-MIB');
```

Comment:
May as well move these two lines into the next `if ($config['discovery_by_ip'] == true) {`